STklos does not compile when there is a stklosrc file

I created a `stklosrc` file in `~/.stklos/stklosrc` with the following content:

```
(display "Hello -- starting STklos")
(newline)
```

When I start STklos, it will (as expected) source the file:

```
$ stklos
Hello -- starting STklos
  \    STklos version 1.60  (Id: 865fea8)
   \   Copyright (C) 1999-2021 Erick Gallesio <eg@unice.fr>
  / \  Université Côte d'Azur
 /   \ [Linux-5.10.0-3-amd64-x86_64/pthread/readline/utf8]
stklos>
```

But strangely, the file interferes with copilation:

```
Making all in lib
make[1]: Entering directory '/home/jeronimo/pkg/scheme/STklos-github/lib'
../src/stklos -l srfis.stk \
                 -f ../utils/update-srfi-list.stk \
                     -- --internal > srfis-data.scm
(cd ..; make SUPPORTED-SRFIS)
make[2]: Entering directory '/home/jeronimo/pkg/scheme/STklos-github'
make[2]: 'SUPPORTED-SRFIS' is up to date.
make[2]: Leaving directory '/home/jeronimo/pkg/scheme/STklos-github'
*** Boot 0
warning: reference to undefined symbol STklos
warning: reference to undefined symbol starting
warning: reference to undefined symbol --
warning: reference to undefined symbol Hello
Compilation time 1276.279ms
*** Boot 1
#[condition &error-message 139945402375616]: variable `Hello' unbound
**** FATAL ERROR: no handler present!
ABORT
make[1]: *** [Makefile:938: ../src/boot.img] Error 1
make[1]: Leaving directory '/home/jeronimo/pkg/scheme/STklos-github/lib'
make: *** [Makefile:423: all-recursive] Error 1
```

Why is it interpreting the string inside the `display` as a list of variables?

--- a/lib/Makefile.am
+++ b/lib/Makefile.am
@@ -315,7 +315,7 @@ srfi-134.ostk: srfi-158.ostk
 # If ./srfis.stk we must rebuild
 #     - srfis-data.scm (used by SRFI0 and features)
 srfis-data.scm: srfis.stk
-	$(STKLOS_BINARY) -l srfis.stk \
+	$(STKLOS_BINARY) -q -l srfis.stk \
 	                 -f ../utils/update-srfi-list.stk \
                      -- --internal > $@
 	(cd ..; $(MAKE) SUPPORTED-SRFIS)

